tests/test_calibrator.py

```diff
@@ -40,36 +40,36 @@ def setUpClass(cls):
         cls.orig_https_context = ssl._create_default_https_context
         ssl._create_default_https_context = ssl._create_unverified_context
 
-    def tearDown(self):
+    def tearDown(self) -> None:
         if os.path.exists("pairs.list"):
             os.unlink("pairs.list")
 
     @classmethod
-    def tearDownClass(cls):
+    def tearDownClass(cls) -> None:
         # TODO: PyTorch has bugs downloading models from their hub on Windows
         ssl._create_default_https_context = cls.orig_https_context
 
-    def calibration_setUp(self, needle, haystack, calibrate_backends):
+    def calibration_setUp(self, needle: str, haystack: str, calibrate_backends: dict[str, str]) -> float:
```

Comment:
Let's just revert this for now, the tests are not checked by `mypy`

Reply:
Done